build.sbt

```diff
@@ -431,9 +445,21 @@ lazy val unidocSettings = Seq(
   ),
   JavaUnidoc / unidoc / unidocAllSources := (JavaUnidoc / unidoc / unidocAllSources).value.map {
     sources =>
-      sources.filterNot { source =>
-        source.toString.contains("$") || source.toString.contains("packageprivate")
-      }
+      sources
+        .filterNot { source =>
+          source.toString.contains("$") || source.toString.contains("packageprivate")
+        }
+        .filter { source =>
+          //
+          // This filter limits javadoc to only files with
+          // api, sapi, or japi directories in their paths.
+          //
+          val str = source.toString
+          str.contains("/udf/") ||
+          str.contains("/api/") ||
+          str.contains("/sapi/") ||
+          str.contains("/japi/")
```

Comment:
Same thoughts as above, note that sapi doesn't exist in javadoc but probably doesn't hurt to keep for consistency.